tests/ft_purescript.vader

```diff
@@ -0,0 +1,48 @@
+Include: include/setup.vader
+
+Execute (pulp parses success message):
+  new
+  file src/Main.purs
+  let output = [
+    \ '* Building project in /home/test/purescript/',
+    \ '{"warnings":[],"errors":[]}',
+    \ '* Build successful.',
+    \ ]
+
+  let result = neomake#makers#ft#purescript#pulp().process_output({'output': output})
+
+  AssertEqual [], result
+
+Execute (pulp parses error message):
+  new
+  file src/Main.purs
+  file src/Other.purs
```

Comment:
Don't call `file` twice on the same buffer, but should call `new` inbetween maybe?!